woocommerce/woocommerce-ios
# Updated KeychainAccess pod from 3.2.0 to 4.2.1, that includes Xcode 12 support

## Description
As part of our Xcode 12 migration, we should update our Pod libraries, because some of them show warnings about the `IPHONEOS_DEPLOYMENT_TARGET` set to iOS 8.0, that is no more supported.
In this PR I updated the KeychainAccess pod from v.3.2.0 to v.4.2.1 that now supports Xcode 12. You can check all the improvements that happened during every bump release [here](https://github.com/kishikawakatsumi/KeychainAccess/releases).

## Testing
- Check that the login process continue to work like before.
- Make sure that this warning disappear:
<img width="500" alt="Schermata 2020-10-16 alle 11 35 42" src="https://user-images.githubusercontent.com/495617/96243092-8c6dee80-0fa4-11eb-98b3-edbe464b6397.png">


Update release notes:
